src/Core/Services/AuthService.cs

```diff
@@ -275,7 +278,7 @@ private async Task<SymmetricCryptoKey> MakePreloginKeyAsync(string masterPasswor
         private async Task<AuthResult> LogInHelperAsync(string email, string hashedPassword, string localHashedPassword,
             string code, string codeVerifier, string redirectUrl, SymmetricCryptoKey key,
             TwoFactorProviderType? twoFactorProvider = null, string twoFactorToken = null, bool? remember = null,
-            string captchaToken = null)
+            string captchaToken = null, string orgId = null)
```

Comment:
Same as the changes to `TokenRequest`, some (but not all) of these changes were later reverted. Please compare to the current diff on the jslib PR.